tests/base/reactor_config/test_reactor_config.py

```diff
@@ -53,6 +53,9 @@ class TestReactorConfigClass:
     def setup_method(self):
         set_log_level("DEBUG")
 
+    def teardown_method(self):
+        set_log_level("INFO")
```

Comment:
Sorry, being nosy, but shouldn't this restore the log level to whatever it was before the test suite was started?

Reply:
you're right yeah! old_loglevel =  get_log_level() etc, could you add that @oliverfunk?